python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -671,13 +625,19 @@ def postprocessoutput(self, outputfiles, outputdir):
                     for currentFile in glob.glob(os.path.join(outputdir, outputfile.namePattern)):
                         os.system("gzip %s" % currentFile)
 
+
             backend_output_postprocess = self.getBackendOutputPostprocessDict()
             if backendClass in backend_output_postprocess:
                 if outputfileClass in backend_output_postprocess[backendClass]:
-                    if backend_output_postprocess[backendClass][outputfileClass] == 'client':
+                    if backend_output_postprocess[backendClass][outputfileClass].lower() == 'client':
                         logger.info("Job %s Putting File %s: %s" % (self.getFQID('.'), getName(outputfile), outputfile.namePattern))
                         outputfile.put()
                         for f in glob.glob(os.path.join(self.outputdir, outputfile.namePattern)):
+                            # FIXME should this be controlled via some per filetype config?
```

Comment:
Yes but where, where do we make sure all other files remove the file apart from the LocalFile, some new function or do we now place the onus on all other filetypes to clean up after themselves. The latter seems like a policy change which needs discussion, whether it's in this PR or later I don't mind.